### PR TITLE
Add PropTypes to components.

### PR DESCRIPTION
**Pull request by skevy**
Thought this might be helpful during development :)

### CODE REVIEW

File: resources/build.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 rm -rf dist/ && mkdir -p dist/ &&
-babel src --ignore __tests__ --out-dir dist/ &&
+babel src --ignore __tests__ --out-dir dist/ --stage=0 &&
```
Comment (asiandrummer) [inline resources/build.sh]: Why was --stage=0 needed?

Comment (skevy) [inline resources/build.sh]: Class properties.

Comment (leebyron) [inline resources/build.sh]: Can we adjust it to only add that one feature so it's more explicit? stage=0 adds a lot of random stuff.

Comment (skevy) [inline resources/build.sh]: Sure! Not sure how I missed the babel settings in package.json. I will add `es7.classProperties` explictly.

**Comment by skevy** [comment]
@leebyron i think that'll do it.

**Comment by leebyron** [comment]
Dope

**Pull request merged**